nixos/tests/gnome-keyring.nix

```diff
@@ -0,0 +1,22 @@
+import ./make-test.nix ({ pkgs, lib, ... }:
+
+with lib;
+
+{
+  name = "gnome-keyring";
+  meta.maintainers = with pkgs.stdenv.lib.maintainers; [ ma27 ];
+
+  nodes.machine = {
+    imports = [ ./common/x11.nix ./common/user-account.nix ];
+    services.xserver.displayManager.auto.user = "bob";
+
+    services.gnome3.gnome-keyring.enable = true;
+    services.gnome3.gnome-keyring.enableService = true;
```

Comment:
It looks a bit weird that a user has to enable both. Maybe gnome should explicitly disable the service, so the user do not has to care about it.